### DISCUSSION TITLE
Which bundler will be the best to use for micro frontend in 2024?

### DISCUSSION THREAD

**Discussion by AsuraKev** [module-federation/module-federation-examples#3281] [General]
Hi all,

throwing out this question as there are many options that can help out building micro frontend. Apart from webpack, there are many new rust based bundler coming out most likely out of experimental in 2024. 

Options:
- vite (originjs/module-federation currently very less actively maintained)
- rspack
- Turborepo 

which one would you recommend worth marry to going forward?

🙂

**Comment by zhoushaw** [comment]
Rspack

**Comment by ScriptedAlchemy** [comment]
Rspack. Hands down. 
It's the same team who owns federation and rspack. We also use federation at the largest known scale - thousands of remotes.
